kura/distrib/src/main/resources/common/customize_snapshot.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2024 Eurotech and/or its affiliates and others
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#  Eurotech
+#
+import sys
+import logging
+import os
+from network_tools import get_eth_wlan_interfaces_names
+
+def main():
+    logging.basicConfig(
+        format='[customize_snapshot.py] %(asctime)s %(levelname)s %(message)s',
+        level=logging.INFO,
+        datefmt='%Y-%m-%d %H:%M:%S',
+        handlers=[
+            logging.StreamHandler()
+        ]
+    )
+    
+    SNAPSHOT_FILENAME = "/opt/eclipse/kura/user/snapshots/snapshot_0.xml"
+    
+    args = sys.argv[1:]
+    if len(args) < 1:
+        logging.info("ERROR: invalid arguments length")
+        exit(1)
+    is_networking_profile = args[0]
+    
+    if is_networking_profile == 'false':
```

Comment:
Comparing a string containing a boolean really irks me. I would start considering the use of `argparse` here:

```suggestion
    parser = argparse.ArgumentParser(
            description="Customize snapshot application")
    parser.add_argument('--networking_profile', action='store_true')
    parser.set_defaults(networking_profile=False)
    
    args = parser.parse_args()
    
    if not args.networking_profile:
```

With `argparse` you also get some niceties like `help` messages and such. Let me know what you think

Reply:
I think that it a bit overkill for this script that it is intended to be used only internally. However, I agree that the boolean parameter management here is not the best :-). I'll implement the `argparse`, so if we'll expand the script in future, it will be alreadt there.